app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  validates :commenter, presence: true
```

Comment:
Delete this line because, in your **Comment table**, you don't have this **commenter** attribute.
[REQUIRED]
![1](https://user-images.githubusercontent.com/58552748/107331121-7e982800-6ac3-11eb-9838-d5c1e952b7fe.png)